test/DatabaseIntegrationTest/Tests/SaveDatabase/SaveUserScenariosBuilder.cpp

```diff
@@ -0,0 +1,104 @@
+#include "stdafx.h"
+#include "SaveUserScenariosBuilder.h"
+
+#include "Tools/Commands/Users/AddUserCommand.h"
+#include "Tools/Commands/Users/EditUserCommand.h"
+#include "Tools/Commands/Users/DeleteUserCommand.h"
+
+#include "SeedCppCoreTestUtilities/Builders/ModelBuilder.h"
+#include "SeedCppCoreTestUtilities/Builders/UserBuilder.h"
+
+
+using namespace seed_cpp::test_utility;
+
+namespace seed_cpp { namespace db_test {
+
+	std::vector<SaveDatabaseTestData> SaveUserScenariosBuilder::build()
+	{
+		// Add some users
+		SaveDatabaseTestData scenarioAddUsers;
+		scenarioAddUsers.setSQLScripts({});
+		scenarioAddUsers.addCommand(std::make_unique<AddUserCommand>(
+			UserBuilder()
+				.setId(std::string("B08BFF46-6AA8-4558-9CD5-0B6EB8C4B768"))
+				.setName("Phil").setSurname("Collins")
+				.setLogin("pcollins").setRole(model::User::ADMIN_ROLE)
+				.setPassword("LeavingNewYorkNeverEasy")
+				.setCreationTime(boost::posix_time::from_iso_string("20150405T112133"))
+				.setUpdateTime(boost::posix_time::from_iso_string("20190523T220314"))
+			.getEntity()
+		));
+		scenarioAddUsers.addCommand(std::make_unique<AddUserCommand>(
+			UserBuilder()
+			.setId(std::string("8294FCCD-4532-4DE5-9DEB-BF81C48378BC"))
+			.setName("Paco").setSurname("Perez Garcia")
+			.setLogin("pperez").setRole(model::User::USER_ROLE)
+			.setPassword("PepeKiko99")
+			.setCreationTime(boost::posix_time::from_iso_string("20180923T130000"))
+			.setUpdateTime(boost::posix_time::from_iso_string("20190521T171400"))
+			.getEntity()
+		));
+
+		return { scenarioAddUsers };
+	}
+
+}}
+
+		// Edit users
+//		ModifyDatabaseData
+//		(
+//			std::string(""),
+//			{
+//				// Lock password
+//				new EditUserCommand(UserDescription(
```

Comment:
What is this commented code for?

Reply:
Same as before: because these tests are not completed/available. I would leave them there as a reference for the moment when they are properly implemented.